hummingbot/client/command/history_command.py

```diff
@@ -86,10 +86,7 @@ async def get_current_balances(self,  # type: HummingbotApplication
         if market in self.markets and self.markets[market].ready:
             return self.markets[market].get_all_balances()
         elif "Paper" in market:
-            paper_balances = global_config_map["paper_trade_account_balance"].value
-            if paper_balances is None:
-                return {}
-            return {token: Decimal(str(bal)) for token, bal in paper_balances.items()}
+            return self.markets[market[:-11]].get_all_balances()
```

Comment:
Yes it's correct. In case of doing paper-trading with binance the market string would be "binance_PaperTrade" but in self.markets the key is just "binance". To remove the "_PaperTrade" suffix the :-11 is necessary. 

But to make this more obvious I will remove the "magic number" and replace it through something like this:
```
paper_trade_suffix = '_PaperTrade'
return self.markets[market[:-len(paper_trade_suffix)]].get_all_balances()
```